packages/public/server/src/module/Ui/templates/entity/view/event.twig

```diff
@@ -22,7 +22,7 @@
 {% set title = 'event' | trans %}
 {% set revision = entity.getCurrentRevision() %}
 {% set content = renderer().toHtml(revision.get('content')) %}
-<article>
+<article id="description">
```

Comment:
why is this `id` needed?

Reply:
For e2e testing. Let's discuss it in https://github.com/serlo/serlo.org/issues/147 and I'll change it afterwards (maybe in a new PR since I also used those testing ID in a last PR).

Reply:
I see. Feel free to check my answer in #147 and let me know if that's okay for you / you want to discuss that.

Reply:
This is not addresses yet, is it? At least it didn't match to what we discussed, if I'm not mistaken. Currently the testing approach is tied tightly to the DOM and even needs changes to production code. Instead of
```.js
const title = await page.evaluate(() => {
  return document.querySelector<HTMLElement>('h1#title')!.innerText
})
expect(title).toBe('Beispielveranstaltung')
```
I'd prefer if we rather do something like
```.js
const { getDocument, queries, wait } = require('pptr-testing-library')

const { getByText } = queries

const $document = await getDocument(page)
const $title = await getByText($document, 'Beispieltext')
// If you also want to assert that it is a heading
console.log($title)
// Something like `expect($title.tagName).toEqual('h1')`
```
Similarly for the other stuff. Then, we don't need changes to our production code and our tests are much more resilient against changes in the DOM (e.g. if we render stuff differently)

Reply:
Oh, I thought I have added a comment here... You would address this in a new PR where I address all current e2e testing refactoring issues. What is better for you? Shall I address the e2e issues in this PR or shall I start a new one, so that this one can be merged (we may create an issue for tracking the remaining tasks)?

Reply:
Follow up issue: #172